atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueService.java

```diff
@@ -1290,32 +1290,29 @@ public void dropTables(final Set<TableReference> tablesToDrop) {
 
     private void dropTablesInternal(final Set<TableReference> tablesToDrop) throws Exception {
         try {
-            clientPool.runWithRetry(new FunctionCheckedException<Client, Void, Exception>() {
```

Comment:
Why are we making a change to `dropTables` in this PR? From the release notes, I thought the only change was to the timestamp methods?